test/api_client/resources/delegates.js

```diff
@@ -55,7 +55,12 @@ describe('DelegatesResource', () => {
 		});
 
 		it('should have methods', () => {
-			return resource.should.have.keys('liskAPI', 'path', 'get', 'getStandby');
+			return resource.should.have.keys(
```

Comment:
This will be sorted after rebasing, right?

Reply:
yes!